eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -362,4 +313,92 @@ function  Get-LatestReleaseDateFromChangeLog
   $changeLogEntries = Get-ChangeLogEntries -ChangeLogLocation $ChangeLogLocation
   $latestVersion = $changeLogEntries[0].ReleaseStatus.Trim("()")
   return ($latestVersion -as [DateTime])
+}
+
+function Sort-ChangeLogEntries {
+  param (
+    [Parameter(Mandatory = $true)]
+    $changeLogEntries
+  )
+
+  try
+  {
+    $changeLogEntries = $ChangeLogEntries.Values | Sort-Object -Descending -Property ReleaseStatus, `
+      @{e = {[AzureEngSemanticVersion]::new($_.ReleaseVersion)}}
+  }
+  catch {
+    LogError "Problem sorting version in ChangeLogEntries"
+    exit(1)
+  }
+  return $changeLogEntries
+}
+
+function Confirm-ChangeLogForRelease {
+  param (
+    [Parameter(Mandatory = $true)]
+    $changeLogEntry,
+    [Parameter(Mandatory = $true)]
+    $changeLogEntries
+  )
+
+  $entries = Sort-ChangeLogEntries -changeLogEntries $changeLogEntries
+
+  $isValid = $true
+  if ($changeLogEntry.ReleaseStatus -eq $CHANGELOG_UNRELEASED_STATUS) {
+    LogError "Entry has no release date set. Please ensure to set a release date with format '$CHANGELOG_DATE_FORMAT'. See https://aka.ms/azsdk/guideline/changelogs for more info."
+    $isValid = $false
+  }
+  else {
+    $status = $changeLogEntry.ReleaseStatus.Trim().Trim("()")
+    try {
+      $releaseDate = [DateTime]$status
+      if ($status -ne ($releaseDate.ToString($CHANGELOG_DATE_FORMAT)))
+      {
+        LogError "Date must be in the format $($CHANGELOG_DATE_FORMAT). See https://aka.ms/azsdk/guideline/changelogs for more info."
+        $isValid = $false
+      }
+
+      $dateHistory = $entries.ReleaseStatus | ForEach-Object { $_.Trim().Trim("()") }
```

Comment:
Do we need to trim everything here? Can't we just do something like `$entries.ReleaseStatus[0] -ne $changeLogEntry.ReleaseSTatus`?